src/ucs/datastruct/ptr_map.h

```diff
@@ -17,21 +17,36 @@
 BEGIN_C_DECLS
 
 
-#define UCS_PTR_MAP_KEY_INVALID    UINT64_MAX
-#define UCS_PTR_MAP_KEY_ID_FLAG    0x1
+/**
+ * The flag is present in key if it is indirect key
+ */
+#define UCS_PTR_MAP_KEY_INDIRECT_FLAG   UCS_BIT(0)
+
+
+/**
+ * The minimal supported pointer alignment
+ */
+#define UCS_PTR_MAP_KEY_MIN_ALIGN       UCS_BIT(1)
 
 
-/* Hash map to find pointer by its key */
+/**
+ * Key to find pointer in @ref ucs_ptr_map_t
+ */
```

Comment:
maybe it should be uintptr_t?

Reply:
it's going to be used in wire protocols

Reply:
wire protocol can be uint64_t, but the structure is local and treats pointers as id